Source/JavaScriptCore/b3/B3Common.h

```diff
@@ -72,9 +73,8 @@ static IntType chillMod(IntType numerator, IntType denominator)
 template<typename IntType>
 static IntType chillUDiv(IntType numerator, IntType denominator)
 {
-    typedef typename std::make_unsigned<IntType>::type UnsignedIntType;
-    UnsignedIntType unsignedNumerator = static_cast<UnsignedIntType>(numerator);
-    UnsignedIntType unsignedDenominator = static_cast<UnsignedIntType>(denominator);
+    auto unsignedNumerator = unsignedCast(numerator);
```

Comment:
Isn't this more of `castToUnsigned` or `castUnsigned` than `unsignedCast`? It's not like cast operation is unsigned.

Reply:
We have `byteCast`, `spanConstCast`, `std::bit_cast` already so it's not that different. Let's see what @darinadler thinks since he suggested the naming in this PR.

Reply:
I see.

Reply:
Maybe `castToUnsigned` is clearer then.

Reply:
FWIW, `unsignedCast` is fine to me given the precedence of other functions.